arxiv/base/urls/tests/test_urlize.py

```diff
@@ -347,3 +355,13 @@ def test_dont_urlize_category_name(self):
                              'category name math.CO should not get urlized')
             self.assertIn('href="http://supermath.co', urlize('supermath.co'),
                           'hostname close to category name should get urlized')
+
+    def test_dont_urlize_arxiv_dot_org(self):
+        with self.app.app_context():
+            urlize = links.urlizer()
+
+            self.assertNotRegex(urlize('somethign https://arxiv.org bla'), r'this.*URL',
+                             'arxiv.org should not get "this https URL" ARXIVNG-2130')
+
+            self.assertNotRegex(urlize('somethign arxiv.org bla'), r'this.*URL',
```

Comment:
```suggestion
            self.assertNotRegex(urlize('something arxiv.org bla'), r'this.*URL',
```

Reply:
Might be nice to have a test that ensures the original text and casing is preserved, e.g.
```python
urlized_arxiv = urlize('on the open-access server arXiv.org we find...')
self.assertNotRegex(urlized_arxiv, r'this.*URL', 'arXiv.org should not get "this https URL" [ARXIVNG-2130]')
self.assertRegex(urlized_arxiv, r'arXiv\.org', 'arXiv.org should be preserved as text [ARXIVNG-2130]')

```